audition/distance_from_best.py

```diff
@@ -33,11 +33,23 @@ def _create(self):
             train_end_time timestamp,
             metric text,
             parameter text,
-            raw_value float,
-            below_best float,
-            below_best_next_time float
+            dist_from_abs_worst float,
```

Comment:
I'm not sure if we can get away without well-defining it, at least by metric (maybe in `catwalk.metrics` as another decoratable attribute). How would you implement 'highest avg of two metrics' with two metrics of opposing directionalities, if you don't know what the absolute value over worst is?